FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/include/retry_utils.h

```diff
@@ -202,6 +202,12 @@ typedef enum RetryUtilsStatus
  */
 typedef struct RetryUtilsParams
 {
+    /**
+     * @brief Max number of retry attempts. Set this value to 0 if the client must
+     * retry forever.
+     */
+    uint32_t maxRetryAttempts;

```

Comment:
I don't think we should be making changes to this header file. Wouldn't it make more sense to keep in sync with the C SDK?

Reply:
The trouble is that the retry mechanism was built with the assumption that it would be only used for reconnection, but I imagine there will be cases where it will be useful to change the number of max attempts. Anyway, I will keep it in sync for now, but I think it is really something to consider. @dan4thewin

Reply:
Wait, so this was added back?

Reply:
@muneebahmed10 Sorry only saw this comment now. Yes, we added it back as it is not a part of any public-facing API that our users consume.